src/blocks/cpu.rs

```diff
@@ -117,7 +117,7 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
             "barchart" => Value::text(barchart),
             "frequency" => Value::hertz(freq_avg),
             "utilization" => Value::percents(utilization_avg * 100.),
-            "max_frequency" => Value::hertz(freqs.iter().copied().max_by(f64::total_cmp).unwrap()),
+            "max_frequency" => Value::hertz(freqs.iter().copied().max_by(f64::total_cmp).unwrap_or_default()),
```

Comment:
I would prefer to be explicit and make `frequency` and `max_frequency` optional (similar to `boost`, with a note in the docs). I.e.

```suggestion
            [if !freqs.is_empty()] "max_frequency" => Value::hertz(freqs.iter().copied().max_by(f64::total_cmp).unwrap()),
```

Reply:
Thanks for the suggestion! I've implemented it and also updated the relevant doc. :tada: